test/functional/Valhalla/src_lw5/org/openj9/test/lworld/ValueTypeSystemArraycopyTests.java

```diff
@@ -110,21 +90,21 @@ public primitive static class SomePrimitiveValueClass2 implements SomeInterface
 	public static SomeIdentityClass[] idArraySrc = new SomeIdentityClass[ARRAY_SIZE];
 	public static SomeValueClass[] vtArrayDst = new SomeValueClass[ARRAY_SIZE];
 	public static SomeValueClass[] vtArraySrc = new SomeValueClass[ARRAY_SIZE];
-	public static SomePrimitiveValueClass[] primitiveVtArrayDst = new SomePrimitiveValueClass[ARRAY_SIZE];
-	public static SomePrimitiveValueClass[] primitiveVtArraySrc = new SomePrimitiveValueClass[ARRAY_SIZE];
+	public static SomeValueClass![] nullRestrictedVtArrayDst = new SomeValueClass![ARRAY_SIZE];
+	public static SomeValueClass![] nullRestrictedVtArraySrc = new SomeValueClass![ARRAY_SIZE];
```

Comment:
`SomeValueClass!`  => `SomeNullRestrictedValueClass!` or whatever new class name will be used based on the above comment. 

The same comment applies to the rest of the file as well.